orchagent/pfcwdorch.h

```diff
@@ -108,7 +109,7 @@ class PfcWdSwOrch: public PfcWdOrch<DropHandler, ForwardHandler>
     const vector<sai_queue_attr_t> c_queueAttrIds;
 
     shared_ptr<DBConnector> m_pfcWdDb = nullptr;
-    shared_ptr<ProducerStateTable> m_pfcWdTable = nullptr;
+    shared_ptr<ProducerTable> m_pfcWdTable = nullptr;
```

Comment:
Why switch to ProducerTable?

Reply:
same questions here.

Reply:
Using statetable, when we want to change a single field, the consumer will receive the state of all the fields and set them one more time. Eg. We set F1 v1. first, and then when set F2 v2 later, consumer can still receive F1 V1 when handling F2 V2 set. This is not ideal as we have the use case to only change the value of a single field. Using statetable will trigger the current syncd error that it only allow to add the plugin with same sha once. 
 
Since we do not frequently change the field value for a single key for flexcounter, there is no benefit to use statetable. So switch to producertable/cosumertable